tests/queries/0_stateless/02843_backup_use_same_s3_credentials_for_base_backup.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+# Tag: no-fasttest - requires S3
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CUR_DIR"/../shell_config.sh
+
+$CLICKHOUSE_CLIENT -nm -q "
+    drop table if exists data;
+    create table data (key Int) engine=MergeTree() order by tuple();
+    insert into data select * from numbers(10);
+"
+
+function write_invalid_password_to_base_backup()
+{
+    local name=$1 && shift
```

Comment:
Why is `shift` here?

Reply:
This is a common practice in bash:

```bash
function foo()
{
    local arg1=$1 && shift
    local arg1=$1 && shift
    ...
}
```

That way you do not rely on the order of args, i.e.

```bash
function foo()
{
    local arg1=$1
    local arg1=$2
    ...
}
```

Here it is not important, but is has zero harm, and later (if) more arguments will be added the diff will contain only one line of change